src/login/components/LoginFormLedger/LoginFormLedger.js

```diff
@@ -42,12 +50,14 @@ export default class LoginFormLedger extends React.Component {
   }
 
   render() {
+    this.getDeviceInfo();
```

Comment:
Is this call intentional?  Is it trying to change state?  Changing state from within a render call is something that should not be done within React, as it can chain re-renders potentially indefinitely.